EIPS/eip-5507.md

```diff
@@ -0,0 +1,121 @@
+---
+eip: 5507
+title: Refundable NFTs
+description: Adds refund functionality to EIP-721 and EIP-1155 NFTs
+author: elie222 (@elie222), Pandapip1 (@Pandapip1)
+discussions-to: https://ethereum-magicians.org/t/eip-5507-refundable-nfts/10451
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-19
+requires: 165, 721, 1155
+---
+
+## Abstract
+
+This EIP adds refund functionality to the [EIP-721](./eip-721.md) and [EIP-1155](./eip-1155.md) standards.
```

Comment:
This abstract seems a bit too terse. I'd like to see a bit more description of the proposal, possibly including (but not limited to) what conditions are required for a refund, and whether they're cooperative/adversarial.